### PR TITLE
New sign in page for membership

### PR DESCRIPTION
**Pull request by svillafe**
This PR implements the new design for membership sign in. 

Screen Shot of the new design:

reg_signin_css

### CODE REVIEW

File: public/signin-page.hbs [hunk 1]
```diff
@@ -23,7 +23,6 @@
 
   {{> components/terms/_terms }}
 
-
```
Comment (SiAdcock) [inline public/signin-page.hbs]: Why no like newline? 😢

**Comment by SiAdcock** [comment]
👍 💯 👍

File: public/signin-page.hbs (hunk 1 above)
Comment (svillafe) [inline public/signin-page.hbs]: I have just updated the code. Nice spot!

**Comment by svillafe** [comment]
@rtyley Can you check the Scala part of this PR?

**Comment by rtyley** [comment]
:+1:

**Comment by svillafe** [comment]
The old design is still working and the new design is behind a switch (clientId=membersB).

signin

**Pull request merged**